Resfeber/Model/DestinationData.swift

```diff
@@ -8,7 +8,6 @@
 
 import Foundation
 
-class Data {
+class DestinationData {
```

Comment:
Is there a reason the destination data is in its own class? If we move the destinations array into the DestinationController class then we can set it as read-only and still use the controller to manage/update the data. I think this would be a safer option and less prone to bugs.

If you agree, feel free to make the change and submit a separate PR.

Reply:
I created the destination data as its own class for when we transition to pulling from the REST API we will need a model (it will have to conform to Codable then but minor changes)